attractions/utils/accepted-file-type.js

```diff
@@ -1,6 +1,6 @@
 /**
  * Validates a File object against an `accept` parameter value.
- * @param {string} references comma-delimited list of reference MIME types (similar to `input`'s `accept` attribute)
+ * @param {?string} references comma-delimited list of reference MIME types (similar to `input`'s `accept` attribute)
```

Comment:
It means just that, `string | null`. Optionality would be `@param {string} [references]`. The thing is that sveld copies the content of the [`@type {}` JSDoc comment](https://jsdoc.app/tags-type.html) exactly as it is, but JSDoc has a syntax slightly different to TypeScript. Of course we can use TypeScript notation everywhere (since it will only be parsed by TS, not JSDoc), but I thought why not just use JSDoc notation? It can help in generating docs later on

Reply:
Idk, to me it looks very strange, and if we were to generate docs, we'd probably have it powered by TypeScript, but whatever